circle.yml

```diff
@@ -66,6 +59,8 @@ jobs:
             sudo apt-get install postgresql-client
             sudo apt-get install unzip
             sudo wget -P /tmp https://releases.hashicorp.com/consul/1.2.1/consul_1.2.1_linux_amd64.zip
+            make install_system_dependencies
```

Comment:
This was not being run before, was it? We might not need it in CI if that's the case.

Reply:
We might not need it. I thought maybe it would be better to have it that way the CI's environment is closer to what we have locally.
I can remove it, tho.

Reply:
Fair enough, let's keep it.